app/src/main/java/com/example/danil/androidcourses/home/HomeActivity.java

```diff
@@ -43,5 +45,14 @@ public void onClick(View v) {
                 startActivity(intent);
             }
         });
+
+        Button btnTask4 = (Button)findViewById(R.id.btn_app_4);
```

Comment:
Lets use Java 8 lambdas for setOnClickListener. Use that links, which I have sent you.